Lib/inspect.py

```diff
@@ -2268,7 +2270,8 @@ def _signature_from_builtin(cls, func, skip_bound_arg=True):
 
 
 def _signature_from_function(cls, func, skip_bound_arg=True,
-                             globals=None, locals=None, eval_str=False):
+                             globals=None, locals=None, eval_str=False,
+                             annotation_format=Format.VALUE):
```

Comment:
you seem to always pass this argument by keyword, and I think it's more readable to do it like that; is it worth enforcing it?

```suggestion
                             *, annotation_format=Format.VALUE):
```